govc/USAGE.md

```diff
@@ -4185,6 +4195,27 @@ Options:
   -t=false               Include sample times
 ```
 
+## namespace.change
+
+```
+Usage: govc namespace.change [OPTIONS]
+
+Modifies an existing namespace on a supervisor cluster. 
```

Comment:
Some terminology fix (here and other places): 
1. the official documentation is "vSphere Namespace" https://docs.vmware.com/en/VMware-vSphere/8.0/vsphere-with-tanzu-concepts-planning/GUID-638682AE-5EC9-47AA-A71C-0BECF9DC27A1.html

2. `supervisor cluster` -> `Supervisor`